pkg/utils/common.go

```diff
@@ -109,3 +111,20 @@ func SanitizeRepoUrl(url string) string {
 
 	return url
 }
+func PrintTable(logger logger.Logger, header []string, rows [][]string) {
```

Comment:
Does this really need a `logger.Logger`, or will any `io.Writer` work?

Reply:
Any io.writer will work

Reply:
The more open the function signature, the more useful this code will be. I suggest we only require the exact interface that is required.